CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change log
 
+## [1.1.0] - 2022-11-21
+
+### Client Driven
+
+#### Changed
+- Updated to Netcode for GameObjects 1.1.0
+- The sample has been refactored to feature a PlayerPrefab inside NetworkManager that is a networked variant of a prefab from [Unity's Starter Assets](https://assetstore.unity.com/packages/essentials/starter-assets-third-person-character-controller-196526). Other improvements include:
```

Comment:
It'd be great if we could discuss some of the bug fixes and improvements to Starter Assets (or link to the doc that does) that users would need to do on their side to make this work.

Reply:
I've created a Jira task [here](https://jira.unity3d.com/browse/MTT-5179) to track this doc.

Reply:
Closed the task, per offline discussion.